packages/extension/src/components/pages/ShareNFT/ShareNFT.tsx

```diff
@@ -58,7 +58,7 @@ export const ShareNFT: FC = () => {
           type: receivingMessage,
           payload: {
             id,
-            nfts: !!messagePayload
+            result: !!messagePayload
```

Comment:
You don't have to wrap in a `result` object here, you can wrap only in the API - Just please keep consistency between all the pages.